pycqed/measurement/waveform_control/pulsar.py

```diff
@@ -1104,6 +1402,38 @@ def __init__(self, name='Pulsar', master_awg=None):
     def get_instance():
         return Pulsar._instance
 
+    def _use_sequence_cache_parser(self, val):
+        if val and not self.use_sequence_cache():
+            self.reset_sequence_cache()
+        return val
+
+    def reset_sequence_cache(self):
+        self._sequence_cache = {}
+        self._sequence_cache['settings'] = {}
+        self._sequence_cache['metadata'] = {}
+        self._sequence_cache['hashes'] = {}
+        self._sequence_cache['length'] = {}
+
+    def check_for_other_pulsar(self):
```

Comment:
Ideally we would also want to detect manual change of the sequences. Could the id be stored in the AWG sequence code?

Reply:
We could consider adding this in the future. You could create an issue to make sure that we'll remember it.